server/server.ts

```diff
@@ -104,6 +86,24 @@ app.post('/api/medications', async (req, res, next) => {
   }
 });
 
+app.get('/api/medications/:userId', async (req, res, next) => {
+  try {
+    const { userId } = req.params;
```

Comment:
You will change this after you add auth. You can keep it, but I don't think it's worth even checking for `userId` right now until you add user management. I would just make it so that it returns all medications. Eventually, this will use the `authMiddleware`, so then you'd get the `userId` from `req.user`.